src/odemis/gui/cont/tabs.py

```diff
@@ -2034,8 +2034,68 @@ def display_new_data(self, filename, data, extend=False):
                                 self.tab_data_model.ar_cal.value)
                 self.tab_data_model.ar_cal.value = u""  # remove the calibration
 
-        # TODO: if all the views are either empty or contain the same streams,
+        # if all the views are either empty or contain the same streams,
         # display in full screen by default (with the first view which has streams)
+        # Show the 1 x 1 view if:
+        # - the streams are all the same in every view 
+        # - views are empty with no projection
+        
+        def compare_projection_lists(list1, list2):
```

Comment:
Usually (by convention), "compare" functions return -1, 0, or 1 ,depending whether the object 1 is less, equal, or more than object 2.
Functions which just returns a boolean for equal or not equal, are usual called "equal" (or "eq" in Python). So I'd suggest to rename this to `projection_list_eq()`.